GithubSessionPrjoect/ContentView.swift

```diff
@@ -14,6 +14,10 @@ struct ContentView: View {
                 .imageScale(.large)
                 .foregroundStyle(.tint)
             Text("Hello, Liver!")
+            Text("Hello, Liver!")
+            Text("Hello, Liver!")
+            Text("Hello, Liver!")
+            Text("Hello, Liver!")
```

Comment:
이 코드는 별로 안좋은 것 같아요.
다음 수정사항은 어떠실까요?
```suggestion
            Text("Hello, Mosu!")
            Text("Hello, Mosu!")
            Text("Hello, Mosu!")
            Text("Hello, Mosu!")
```

Reply:
다음에 고쳐보도록 하겠습니다 ㅠㅠ